libs/sdk-core/src/persist/transactions.rs

```diff
@@ -1,16 +1,34 @@
 use super::db::SqliteStorage;
-use crate::models::{Payment, PaymentDetails};
-use crate::models::{PaymentType, PaymentTypeFilter};
+use crate::lnurl::pay::model::SuccessActionProcessed;
+use crate::models::*;
 use anyhow::{anyhow, Result};
 use rusqlite::types::{FromSql, FromSqlError, ToSql, ToSqlOutput};
 use rusqlite::OptionalExtension;
 use rusqlite::Row;
 use std::str::FromStr;
 
 impl SqliteStorage {
+    /// Inserts payments into the payments table.
+    ///
+    /// If a payment has details of type [LnPaymentDetails] which contain a [SuccessActionProcessed],
+    /// then it is inserted in the separate SuccessActions table.
+    ///
+    /// The [SuccessActionProcessed] can be inserted separately via [SqliteStorage::insert_lnurl_success_action],
+    /// if it's not part of the payment when this method is called.
     pub fn insert_payments(&self, transactions: &[Payment]) -> Result<()> {
+        // Insert available SuccessActions in separate table
+        for p in transactions.iter() {
```

Comment:
I don't think we need this. The success action is inserted after succesfull payment in a different insert.

Reply:
Done